src/eva/tests/config/testMonSummary.yaml

```diff
@@ -12,12 +12,12 @@ datasets:
     sensor: hirs4
     control_file:
       - ${data_input_path}/time.hirs4_metop-a.ctl
-    filenames:
-      - ${data_input_path}/time.hirs4_metop-a.2015051418.ieee_d
-      - ${data_input_path}/time.hirs4_metop-a.2015051500.ieee_d
-      - ${data_input_path}/time.hirs4_metop-a.2015051506.ieee_d
-      - ${data_input_path}/time.hirs4_metop-a.2015051512.ieee_d
-      - ${data_input_path}/time.hirs4_metop-a.2015051518.ieee_d
+    filenames_template:
+        template: ${data_input_path}/time.hirs4_metop-a.{datetime}.ieee_d
```

Comment:
What if I want to do something like this:
```
template: gdas.%Y%m%d/%H/atmos/gdas.t%Hz.atmanl.nc
```
Would this work or does it only insert it in one spot in the filename?

Reply:
That is a good point. I updated the code to not take a format and rather just rely on the formatting in the template. Easier and less ambiguous for the user.